src/main/webapp/resources/js/userCalendar.js

```diff
@@ -951,10 +953,23 @@ function renderCalendar(objects, id, workingHoursStart, workingHoursEnd) {
                     $('#eventTitle').html(calEvent.title);
                     $('#startTime').html('Start at :' + '<b>'+ calEvent.start.format('HH:mm'));
                     $('#endTime').html('Ends at :' + '<b>'+calEvent.end.format('HH:mm'));
+
+                    textArea.css({"height": parseInt(eventInfo.css('height')) - 100,
+                        "width": parseInt(eventInfo.css('width')) - 30,
+                        "resize": "none"});
+
+                    $('#eventDescription').html('Description:');
                     if (calEvent.description != "") {
-                        $('#eventDescription').html('<b>Description : </b><br>' + eventDescription);
+                        textArea.val(eventDescription);
+                    } else {
+                        textArea.val("This event hasn't any description");
```

Comment:
i'd change textArea to some tag likr div..
maybe it's even better to make a condition like:

if (eventDescription.length> 300 )  -> textArea.val(eventDescription); textArea.show();
else textArea.hide and show common div with your short description.

so it'll looks like: if you have short description then it'll be shown like it was before just without breaking borders ;) and if you have long description - then show it in textArea